common/app/model/meta.scala

```diff
@@ -295,8 +508,9 @@ trait Elements {
 /**
  * Tags lets you extract meaning from tags on a page.
  */
-trait Tags {
-  def tags: Seq[Tag] = Nil
+final case class Tags(
+  tags: Seq[Tag]) {
```

Comment:
yeah that would be sensible, would reduce the number of `tags.tags` lines. There's plenty of other tidy up too, (like the filenames in the model folder), but I can't make this PR any bigger!